components/ProgrammingProblem.jsx

```diff
@@ -8,7 +8,7 @@ const ProgrammingProblem = ({ problem, index, id }) => {
     <Link
       href={"/user/problems/" + id}
       className={`${
-        index % 2 == 0 ? "" : "bg-code-darkerpurple"
+        index % 2 == 0 ? "h-20" : "bg-code-darkerpurple h-20"
```

Comment:
if both require it, then u can move the h-20 outside, also the height should be done by margin and padding since there is content inside alr